### PR TITLE
Fix sed syntax

### PR DESCRIPTION
**Pull request by aaron-lane**
## Background

This branch fixes the sed syntax that replaces the module source.

Relates #122 

## How Has This Been Tested

To be tested in #120.

## This PR makes me feel

optional gif describing your feelings about this pr

### CODE REVIEW

File: .github/workflows/handler-destroy.yml [hunk 1]
```diff
@@ -32,9 +32,10 @@ jobs:
       - name: Set Terraform Module Source
         working-directory: ${{ env.WORK_DIR_PATH }}
         env:
-          REPO_FULL_NAME: ${{ github.event.client_payload.pull_request.head.repo.full_name }}
-          REF: ${{ github.event.client_payload.pull_request.head.sha }}
-        run: sed --in-place='s/source = "..\/.."/source = "github.com\/$REPO_FULL_NAME?ref=$REF"/' main.tf
+          LOGIN: ${{ github.event.client_payload.pull_request.head.repo.owner.login }}
+          NAME: ${{ github.event.client_payload.pull_request.head.repo.name }}
+          SHA: ${{ github.event.client_payload.pull_request.head.sha }}
+        run: sed --in-place "s/source = \"..\/..\"/source = \"github.com\/$LOGIN\/$NAME?ref=$SHA\"/" main.tf
```
Comment (sudomateo) [inline .github/workflows/handler-destroy.yml:38]: Do you have an example of what you want your before and after text to look like here?

Based on what I'm seeing it looks like this regular expression will turn this:

```
source = "../../"
```

Into this:

```
source = "github.com/$LOGIN/$NAME?ref=$SHA"
```

However, it looks like this:

```
source = "../../mymodule"
```

Would turn into this:

```
source = "github.com/$LOGIN/$NAME?ref=$SHA/mymodule"
```

**Review by sudomateo — COMMENTED** [review]
Waiting on an answer to a clarifying question.

File: .github/workflows/handler-destroy.yml (hunk 1 above)
Comment (aaron-lane) [reply .github/workflows/handler-destroy.yml:38]: Your first example is what we're doing here. The second scenario won't come in to play as the logic only matches against `source = "../.."` (and the test currently does not instantiate any other modules).

Comment (sudomateo) [reply .github/workflows/handler-destroy.yml:38]: Awesome. This looks good then.

If in the future you wanted to expand things to account for `../..`, `../../`, or `../../mymodule` then this regex:

```
"s/source\s*=\s*\"\.\.\/\.\.\/\?.*\"/source = \"github\.com\/$LOGIN\/$NAME\?ref=$SHA\"/"
```

Would turn this:

```
source = "../.."
source = "../.."
source = "../../mymodule"
source="../../mymodule"
```

Into this:

```
source = "github.com/hashicorp/tfe-module?ref=123456"
source = "github.com/hashicorp/tfe-module?ref=123456"
source = "github.com/hashicorp/tfe-module?ref=123456"
source = "github.com/hashicorp/tfe-module?ref=123456"
```

**Review by sudomateo — APPROVED** [review]

**Pull request merged**